test/models/validator_test.rb

```diff
@@ -47,37 +77,38 @@ def setup
   end
 
   test '#private_validator? returns true if validator\'s commission is 100%' do
-    validator = create(:validator, network: 'mainnet')
-    assert create(:validator_score_v1, commission: 100, validator: validator).validator.private_validator?
-    refute create(:validator_score_v1, commission: 99, validator: validator).validator.private_validator?
+    @validator.update(network: 'mainnet')
```

Comment:
perhaps it's better to create separate validator as it can potentially impact other tests

Reply:
`setup` method is called before every test, so the initial validator is created and the changes from other tests are rollbacked. The easiest way to see it is to put ie. `puts @validator.network` at the bottom of the setup method and run tests.